components/Wizard/CreateImageWizard.js

```diff
@@ -5,9 +5,15 @@ import { Button } from "@patternfly/react-core";
 import { startCompose, fetchingComposeTypes } from "../../core/actions/composes";
 
 import ImageCreator from "./ImageCreator";
-import { imageOutput, details, review } from "./steps";
+import { imageOutput, details, packages, review } from "./steps";
 import "./CreateImageWizard.css";
 
+import BlueprintApi from "../../data/BlueprintApi";
+
+import * as composer from "../../core/composer";
+import { setBlueprint } from "../../core/actions/blueprints";
```

Comment:
This is unused. Did you intend on setting the blueprint before committing?

Reply:
Oof, good catch.